content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -34,8 +34,13 @@ The operator is beta in 1.6 and suitable for production use.
     - A service to access operator metrics
     - Necessary Istio operator RBAC rules
 
-    See the available `istioctl operator init` flags to control which namespaces the controller and Istio are installed
-    into and the installed Istio image sources and versions.
+    You can also pass single or multiple namespaces to watch by using `--watchedNamespaces` flag.
```

Comment:
Why does just this one flag need an example? What namespaces does it watch by default?

Reply:
If there are no namespaces provided to watch, by [default it will watch](https://github.com/istio/istio/pull/24696/files#diff-76640f501d7b88f49ad1adbe61517bdcR5) `istio-system`.

Reply:
This flag needs an example because it has created confusion for users. See next comment.